bin/acm_with_dns_validation.json

```diff
@@ -47,10 +47,10 @@
             "  }",
             "  ",
             "  if (event.RequestType !== 'Create') {",
-            "    return cloudFormationResponseHandler.send(event, context, cloudFormationResponseHandler.SUCCESS);",
+            "    cloudFormationResponseHandler.send(event, context, cloudFormationResponseHandler.SUCCESS);",
```

Comment:
I almost did the same thing when I was trying to change this yesterday, you can't change this to not have the return unless there are some either `else if` or `return;` statements after the line otherwise both this and the next line will be executed.

Reply:
Sigh. Looks like we'll have to invest more time on that. You on it?

Reply:
potentially. would you prefer a macro, it is likely where I would prefer to invest time.